packages/cli/README.md

```diff
@@ -1,5 +1,119 @@
 # @code-pushup/cli
 
-**TODO:**
+**Quality metrics for web projects, highly configurable.**
```

Comment:
Maybe also like above suggested:

> Wrap your favourite tool and integrate it into your CI, visualise your results in a beautiful dashboard

Reply:
Yes, these should all be the same tag line.